sdk/master_board_sdk/src/master_board_interface.cpp

```diff
@@ -347,6 +347,12 @@ void MasterBoardInterface::ParseSensorData()
     imu_data.linear_acceleration[i] = D16QN_TO_FLOAT(sensor_packet.imu.linear_acceleration[i], IMU_QN_ACC);
   }
 
+  /*Read Power Board data*/
+  powerboard_data.current_bus = sensor_packet.powerboard.vshunt * 5e-6 / 4e-3; 
+  powerboard_data.voltage_bus = sensor_packet.powerboard.vbus * 3.125e-3;
+  powerboard_data.energy_bus = sensor_packet.powerboard.energy * 3.125e-3 * 5e-6 / 4e-3 ;
```

Comment:
Two questions:
- should the constants used here be documented somewhere?
- small comment: I was not sure what `vshunt` stands for until reading this line and figuring out it means the electric current pulled by the motor board. Is `vshunt` the technical term or would it make sense to rename it to `cbus` or similar to make it more clear that it's about the current on the bus? This is only a small comment, so feel free to ignore it.